lib/ClimaLandSimulations/src/utilities/timestepper_setup.jl

```diff
@@ -1,7 +1,32 @@
-N_spinup_days = 30
-N_days = N_spinup_days + 30
-tf = Float64(t0 + 3600 * 24 * N_days)
-t_spinup = Float64(t0 + N_spinup_days * 3600 * 24)
-saveat = Array(t_spinup:(n * dt):tf)
-timestepper = CTS.RK4()
-ode_algo = CTS.ExplicitAlgorithm(timestepper)
+export timestepper_setup
+
+"""
+    timestepper_setup(site_setup_out;
+        N_spinup_days = 30,
+        N_days_sim = 30,
+        timestepper = CTS.RK4(),
+        ode_algo = CTS.ExplicitAlgorith(timestepper)
+        )
+
+Define the setup for the simulation timestepper. 
```

Comment:
Could you explain the default timestepper (4th order Runge Kutta method), and that other timesteppers from ClimaTimeSteppers could be used? Also maybe explain what happens during spinup vs sim days